src/Tribe/Cache.php

```diff
@@ -309,7 +311,7 @@ public function make_key( $components, $prefix = '', $sort = true ) {
 	 * @since 5.0.0
 	 */
 	public function offsetExists( $offset ) {
-		return in_array( $offset, $this->non_persistent_keys );
+		return isset( array_flip( $this->non_persistent_keys )[ $offset ] );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: isset() must only contain variables (PHP 5.6 compatibility)

<sub>hash: 522413556a8bd6b1d08db82c6d92cf18</sub>